src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -241,7 +241,13 @@ public void OnStopActivity(Activity activity, object payload)
                     context.GetEndpoint() as RouteEndpoint)?.RoutePattern.RawText;
             if (!string.IsNullOrEmpty(routePattern))
             {
-                activity.DisplayName = GetDisplayName(context.Request.Method, routePattern);
+                // only override the display name if it was not set by the user
```

Comment:
@PascalSenn - Could you add a unit test for this? These tests are more focused on `http.route` tag value.

Reply:
You mean instead of the one in RoutingTests?

Reply:
ok to add this in addition to unit test. @alanwest - Do you have any preference?

Reply:
I think it makes sense to test this new behavior separate from the tests for http.route.